### ISSUE TITLE
Call to undefined function Q

### ISSUE THREAD

**Issue by joshp23** [conversejs/roundcube-plugin#7]
When I attempt to load the user interface preferences in RC, it just hangs. The error log gives this:
```
[02-Oct-2018 17:09:35 America/New_York] PHP Fatal error:  Uncaught Error: Call to undefined function Q() in /var/www/webmail/plugins/converse/converse.php:270
Stack trace:
#0 /var/www/webmail/program/lib/Roundcube/rcube_plugin_api.php(440): converse->preferences_list(Array)
#1 /var/www/webmail/program/steps/settings/func.inc(1265): rcube_plugin_api->exec_hook('preferences_lis...', Array)
#2 /var/www/webmail/program/steps/settings/edit_prefs.inc(28): rcmail_user_prefs('general')
#3 /var/www/webmail/index.php(303): include_once('/var/www/webmai...')
#4 {main}
  thrown in /var/www/webmail/plugins/converse/converse.php on line 270
```

**Comment by mckaygerhard** [comment]
its the translation of the title in settings, this only happened in the very older version.. in that case just go around line 275 and set "title" to "converse loading", or just change `Q($this-gettex('enableprebind'))` to "auto" or any string so will be `html::label(Q($this-gettex('enableprebind')))` erase to a just a simple string

**Comment by drlight17** [comment]
old version issue. closed.

**Comment by mckaygerhard** [comment]
fixed with PR #3 thanks
